src/mackerel-container-agent-definition.ts

```diff
@@ -22,6 +22,7 @@ export interface Props
   ignoreContainer?: string
   hostStatusOnStart?: MackerelHostStatus
   generation?: AmazonLinuxGeneration
+  image?: ContainerImage
```

Comment:
oops, now we can remove this type-level trick: https://github.com/aereal/cdk-mackerel-container-agent/pull/22/files#diff-04e55e26bda6d3f4772a48e1a6d59c53R18

We'd do this for accepting all of ContainerDefinitionProps except of `image`, but now we accept `image` too.

Reply:
😱 && 🆗

Reply:
Hmm, current situation is following:
- Original ContainerDefinitionProps has `image` as mandatory
- we (or me? ;-)) want to redefine `image` as optional

In this case we still have to pick properties except `image` and redefine it as optional `image?`, maybe?

Reply:
I see. You can use [`Partial` type-level function](https://github.com/Microsoft/TypeScript/blob/9e28a811c5670ebfd233d592c936d3b7afcd7b8f/lib/lib.es5.d.ts#L1424) in this case.